waku/v2/node/waku_node.nim

```diff
@@ -148,6 +148,13 @@ proc new*(T: type WakuNode,
           nameResolver: NameResolver = nil,
           sendSignedPeerRecord = false,
           dns4DomainName = none(string),
+          discv5Enabled = false,
+          discv5TableIpLimit = none(uint),
+          discv5BucketIpLimit = none(uint),
+          discv5BitsPerHop = none(int),
```

Comment:
Hard to constrain these types more, since the upstream lib doesn't expose types like `Discv5TableIpLimit`